packages/vite/src/template-tag.ts

```diff
@@ -19,26 +19,32 @@ export function templateTag({ inline_source_map } = { inline_source_map: false }
       if (options.custom?.embroider?.isExtensionSearch) {
         return null;
       }
-      let resolution = await this.resolve(id, importer, {
-        skipSelf: true,
-      });
-      if (resolution) {
-        return resolution;
-      }
-      for (let candidate of candidates(id)) {
-        resolution = await this.resolve(candidate, importer, {
+      if (id.endsWith('.gjs') || id.endsWith('.gts')) {
+        let resolution = await this.resolve(id, importer, {
           skipSelf: true,
-          custom: {
-            embroider: {
-              isExtensionSearch: true,
-              enableCustomResolver: false,
-            },
-          },
         });
         if (resolution) {
-          return {
-            id: resolution.id,
-          };
+          return resolution;
+        }
+      }
+      for (let candidate of candidates(id)) {
+        try {
+          let resolution = await this.resolve(candidate, importer, {
```

Comment:
in try catch because it can fail during resolving: 
`[commonjs--resolver] Missing "./modulepreload-polyfill.gjs" specifier in "vite" package`